Fabric.Databus.API/Configuration/AppConfiguration.cs

```diff
@@ -1,9 +1,11 @@
-﻿using Fabric.Databus.Domain.Configuration;
-
-namespace Fabric.Databus.API.Configuration
+﻿namespace Fabric.Databus.API.Configuration
 {
-    public class AppConfiguration : IAppConfiguration
-    {
-        public ElasticSearchSettings ElasticSearchSettings { get; set; }
-    }
+		public class AppConfiguration : IAppConfiguration
+		{
+				public Domain.Configuration.ElasticSearchSettings ElasticSearchSettings { get; set; }
+				public string Authority { get; set; }
```

Comment:
You may want to consider encapsulating the IdentityServer settings in another class.

Reply:
This was intentional - there's no way to pass scopes via an env var in docker if I use the suggested encapsulating class as it requires the scopes to be an array. That said, I suppose I could use an arbitrary class instead of the IdentityServerConfidentialClientSettings class.

Reply:
The scopes are something that needs to be specified at design time anyway since you will authoriae against them in your modules, so you could place those in the appsettings.json, and only use docker environment variables for things that change from environment to environment, like the Authority.